apps/mobile-wallet/src/App.tsx

```diff
@@ -171,11 +185,9 @@ const Main = ({ children, ...props }: ViewProps) => {
   useInterval(refreshAddressDataWhenPendingTxsConfirm, 5000, pendingTxs.length === 0)
 
   return (
-    <RootSiblingParent>
```

Comment:
This seems to only have been needed because of the `react-native-root-toast` library: https://www.npmjs.com/package/react-native-root-toast#install